src/integer/poly_over_z/get.rs

```diff
@@ -49,6 +49,24 @@ impl GetCoefficient<Z> for PolyOverZ {
     }
 }
 
+impl PolyOverZ {
+    /// Returns the degree of a polynomial [`PolyOverZ`] as a [`i64`].
+    /// The zero polynomial has degree '-1'.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::integer::PolyOverZ;
+    /// use std::str::FromStr;
+    ///
+    /// let poly = PolyOverZ::from_str("4  0 1 2 3").unwrap();
+    ///
+    /// let degree = poly.get_degree(); // This would only return 3
+    /// ```
+    pub fn get_degree(&self) -> i64 {
+        unsafe { fmpz_poly_degree(&self.poly) }
+    }
+}
+
```

Comment:
I would open the discussion whether this should be in get.rs or in properties.rs. We have introduced properties.rs for some other types, I personally think, that it could fit into both, what do you think?
Probably get is fine, but lets just briefly talk about it to make it consistently

Reply:
alternatively you could have considered adding length, which returns the number of coefficients, was this a purposeful decision to choose degree over length? Because I think that calling the length function within the norms would make the code readable a little better

Reply:
what do you think about the second question?

Reply:
for now i just wanted to implement on of the two functions and get_degree is more intuitive to use by the end user